### DISCUSSION TITLE
Paligemma

### DISCUSSION THREAD

**Discussion by tucnak** [ggml-org/llama.cpp#7296] [Ideas]
https://huggingface.co/blog/paligemma

So, now that Google has released Paligemma (which is SigLip, as opposed to CLIP-based) what would it take to support it similarly to Gemma, and LLaVA? I will be benching it against both gemma-2b (on text tasks) and 7b llava (on vision tasks) soon enough to get some idea where it sits, but God it's annoying to get transformers working on macOS, and reliably, too. Llama.cpp support would bring additional advantage of bringing it to Android/iOS where Llava is quite prohibitive.

**Comment by lewiswatson55** [comment]
Have you had much luck getting paligemma running? Would love to see it added to Ollama too :)
